src/Symfony/Bundle/FrameworkBundle/Resources/config/schema/symfony-1.0.xsd

```diff
@@ -238,6 +238,7 @@
         <xsd:sequence>
             <xsd:element name="marking-store" type="marking_store" />
             <xsd:element name="supports" type="xsd:string" minOccurs="1" maxOccurs="unbounded" />
+            <xsd:element name="initial_place" type="xsd:string" />
```

Comment:
should be `<xsd:element name="initial_place" type="xsd:string" minOccurs="0" maxOccurs="1" />`